gravitee-apim-console-webui/src/management/api-score/dashboard/api-score-dashboard.component.html

```diff
@@ -157,7 +157,7 @@ <h1 class="page-title">API Score</h1>
         <ng-container matColumnDef="score">
           <th mat-header-cell *matHeaderCellDef id="score">Score</th>
           <td mat-cell *matCellDef="let element">
-            @if (element.score == null) {
+            @if (element.score == null || element.score === -1) {
```

Comment:
```suggestion
            @if (element.score == null || element.score < 0) {
```

to handle more possible future cases